common/src/main/java/com/microsoft/hydralab/performance/parsers/AndroidMemoryInfoResultParser.java

```diff
@@ -20,6 +20,7 @@
 
 public class AndroidMemoryInfoResultParser implements PerformanceResultParser {
     private static final int MEM_INFO_LENGTH = 19;
+    private static final String SUMMARY_DESCRIPTION = "Android memory info summary";
```

Comment:
We are planning to show the description for every dot, and this will be shown on the chart label. So, let's keep it in Java code. We will update the description with test task info in the future.